src/Optimizers/WebpOptimizer.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ImageCompression\Optimizers;
+
+use Exception;
+use ImageCompression\ImageOptimizerInterface;
+use WebPConvert\WebPConvert;
+
+class WebpOptimizer implements ImageOptimizerInterface
+{
+    const WEBP_EXT = 'webp';
+
+    public function optimizeImage(string $pathToImage, string $pathToOutput = null)
+    {
+        try {
+            $pathToImageLevel = $this->getWebpPath($pathToOutput);
+
+            $options = [
```

Comment:
Podemos borrar los commets y poner: 
```suggestion
            /** 
             * @link https://github.com/rosell-dk/webp-convert/blob/master/docs/v2.0/converting/introduction-for-converting.md#configuring-the-options
             */
            $options = [
```

o directamente 

```php 
/** 
 * @link https://github.com/rosell-dk/webp-convert
 */ 
class WebpOptimizer implements ImageOptimizerInterface

Reply:
Listo!